app/views/templates/_form.html.erb

```diff
@@ -98,13 +98,15 @@
     <div class="tab-pane" id="help">
       <div>
         <h1><%= _('Template syntax') %></h1>
-        <%= _('Provisioning templates are generated using the Embedded Ruby (ERB) language.') %>
+        <%= _('%s are generated using the Embedded Ruby (ERB) language.') % @resource_class.model_name.human.pluralize %>
         <%= _('The following table and example snippet shows global methods, variables and class methods and how to use them.') %>
         <%= _('These are guaranteed to work via the safe mode rendering, to ensure a template can do nothing harmful.') %>
         <% unless Setting[:safemode_render] -%>
           <%= _('To enable safe mode, navigate to Settings page and enable Safemode rendering option.') %>
         <% end -%>
-        <%= link_to(_('For more info visit our documentation.'), documentation_url(controller.documentation_anchor)) if controller.respond_to?(:documentation_anchor) %>
+        <%= (_'For more info visit ') %>
+        <%= (link_to(_('our documentation'), documentation_url(controller.documentation_anchor)) + ' and ') if controller.respond_to?(:documentation_anchor) %>
+        <%= link_to(_('the DSL (Domain Specific Language) documentation.'), ApipieDSL.configuration.doc_base_url) %>
```

Comment:
This is not the right way to translate. You should really use interpolation in strings but I'm not sure how to deal with links in translated strings.

Reply:
Thanks. Please have a look if its now better.